README.md

```diff
@@ -71,1043 +25,59 @@ Table of Contents
             * [Running performance tests](#running-performance-tests)
             * [List of test tags](#list-of-test-tags)
          * [Running the Kogito Operator locally](#running-the-kogito-operator-locally)
-      * [Contributing to the Kogito Operator](#contributing-to-the-kogito-operator)
 
 Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
```

Comment:
The TOC is not being generated in the preview. Not sure if it's a GH bug or not, but have you generated the TOC using https://github.com/ekalinin/github-markdown-toc tool?

Reply:
@ricardozanini Not sure since it appears correctly in the file preview for me. I didn't run the toc tool because it was a matter of deleting a lot and then just tweaking like one line, so I just updated the TOC manually. But I've just generated it using the toc tool and pasted the new TOC to be sure. I confirmed it looks good in the file preview and in my branch in GH. If you still see an issue on your end, then something else is up.